zipline/finance/performance/position_tracker.py

```diff
@@ -27,6 +28,140 @@
 log = logbook.Logger('Performance')
 
 
+PositionStats = namedtuple('PositionStats',
+                           ['net_exposure',
+                            'gross_value',
+                            'gross_exposure',
+                            'short_value',
+                            'short_exposure',
+                            'shorts_count',
+                            'long_value',
+                            'long_exposure',
+                            'longs_count',
+                            'net_value'])
+
+
+def calc_position_values(amounts,
+                         last_sale_prices,
+                         value_multipliers):
+    iter_amount_price_multiplier = zip(
+        amounts,
+        last_sale_prices,
+        itervalues(value_multipliers),
+    )
+    return [
+        price * amount * multiplier for
+        price, amount, multiplier in iter_amount_price_multiplier
+    ]
+
+
+def calc_net_value(position_values):
+    if len(position_values) == 0:
```

Comment:
I agree that they should be in different commits.I'm not sure I buy the benefit of separating out a change like this into a separate PR.  Follow-ups have a tendency to either not be followed-up on, or else they sit around unreviewed forever.  I'm increasingly of the opinion that the only way our codebase ever improves is if we actively refactor it while we're working on features.